job_templates/ansible_roles_-_install_from_git.erb

```diff
@@ -23,6 +28,7 @@ model: JobTemplate
 ---
 - hosts: all
   tasks:
-    - command: git clone <%= input('git_repository') %> <%= input('location') %>
-      register: out
-    - debug: var=out
+    - git:
+        repo: <%= input('git_repository') %> 
+        dest: <%= input('location') %>
+        force: <%= input('force').present? ? input('force') : 'no' %>
```

Comment:
We could just paste input('force') here if you decide to specify options, see above